### ISSUE TITLE
Can we trigger event during close image picker modal 

### ISSUE THREAD

**Issue by LSmint** [NativeScript/nativescript-imagepicker#177]
I would like to perform some script after user tapped the modal close button, can we add Promise reject function to return a catch result during trigger the cancel event?

In imagepicker.ios.js file:
```
ImagePickerControllerDelegate.prototype.qb_imagePickerControllerDidCancel = function (imagePickerController) {
        this._reject("cancelled"); <--- add something like this
        imagePickerController.dismissViewControllerAnimatedCompletion(true, null);
    };
```

**Comment by zbranzov** [comment]
The fix is released in v6.0.1 of the plugin. Enjoy!

**Comment by lucasumberto** [comment]
Hi!

On iOS, when the picker is dismissed by native gesture (pushing it down) the close event is not fired like the close button. Would be nice if it fires this event too.
